### PR TITLE
Get access rules test and test for adding access rule to non existent bucket

### PR DESCRIPTION
**Pull request by adfost**
Added the increment to this PR:

Screen Shot 2022-05-24 at 7 55 04 PM

### CODE REVIEW

File: integration/access_rules_test.go [hunk 1]
```diff
@@ -94,3 +94,61 @@ func Test_AddAccessRuleAPI(t *testing.T) {
...
+			},
+			expectedStatus: 200,
+			expectedError:  nil,
+		},
+		{
+			name: "Get Access Rule - Invalid",
+			args: args{
+				bucket: "testaccessruleget2",
+			},
+			expectedStatus: 200,
```
Comment (dvaldivia) [inline integration/access_rules_test.go:125]: invalid should not return `200`, it should return `404`

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: integration/access_rules_test.go (hunk 1 above)
Comment (adfost) [reply integration/access_rules_test.go:125]: ok we just showed the code is broken. I can fix it.

**Review by cniackz — CHANGES_REQUESTED** [review]
Also remember to squash the commits from many to one 👍

**Review by cniackz — APPROVED** [review]

**Review by cniackz — APPROVED** [review]
LGTM

**Review by cniackz — APPROVED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**